docker-compose.yml

```diff
@@ -177,15 +177,44 @@ services:
       timeout: 20s
       retries: 10
 
+  export:
+    image: lisk/service_export
+    depends_on:
+      - redis_common
+      - redis_export
+    networks:
+      - services_network
+      - export_network
+    environment:
+      - SERVICE_BROKER=redis://redis_common:6379/0
+      - SERVICE_MARKET_REDIS=redis://redis_market:6379/0
+      - SERVICE_MARKET_FIAT_CURRENCIES=${SERVICE_MARKET_FIAT_CURRENCIES}
```

Comment:
This lines are not needed here:
 - SERVICE_MARKET_TARGET_PAIRS=${SERVICE_MARKET_TARGET_PAIRS}
 - SERVICE_MARKET_TARGET_PAIRS=${SERVICE_MARKET_TARGET_PAIRS}